src/vm/codeman.cpp

```diff
@@ -2486,12 +2541,24 @@ void* EEJitManager::allocCodeRaw(CodeHeapRequestInfo *pInfo,
         if (mem == NULL)
             ThrowOutOfMemory();
         _ASSERTE(mem);
+#if defined(FEATURE_JIT_DROPPING)
+        if (bUseCachedJitDroppedCodeHeap) {
+            pCodeHeap->SetHeapFull();
+            pCodeHeap->SetHeapFullForJumpStubs();
+        }
+#endif
     }
     
     if (bUseCachedDynamicCodeHeap)
     {
         pInfo->m_pAllocator->m_pLastUsedDynamicCodeHeap = pCodeHeap;
     }
+#if defined(FEATURE_JIT_DROPPING)
+    else if (bUseCachedJitDroppedCodeHeap)
+    {
+        pInfo->m_pAllocator->m_pLastUsedJitDroppedCodeHeap = pCodeHeap;
```

Comment:
I believe that analogy to the original code is not needed here since it cannot be reused for other methods, while it's already used for one method...